src/containers/Node/NodeStructure/NodeStructure.tsx

```diff
@@ -77,13 +65,13 @@ function NodeStructure({nodeId, className}: NodeStructureProps) {
     }, [nodeId, dispatch]);
 
     useEffect(() => {
-        if (!isEmpty(nodeStructure) && scrollContainer) {
+        if (!isEmpty(nodeStructure) && scrollContainerRef.current) {
```

Comment:
Returned old behaviour, where `scrollContainerRef.current` wasn't in dependency array. This case is the same, but doesn't cause linter warnings